test/unit/presenters/publishing_api/world_location_news_page_presenter_test.rb

```diff
@@ -30,6 +30,7 @@ def world_location
             href: newer_link.url,
           },
         ],
+        mission_statement: "This is a great mission", #TODO I don't think this ever actually can be nil in practice (although it's nillable in db schema. How to handle)
```

Comment:
I will remove this comment as decided to handle nil cases. I could only find two in the database, and I think they are old. Ideally it'd be nice for schemas to have fewer nullable fields (I think the current UI allows for blank mission statements, but these are represented as empty strings in the database) but that's beyond the scope of this.